build.sbt

```diff
@@ -500,6 +500,10 @@ def akkaModule(name: String): Project =
     .settings(akka.AkkaBuild.buildSettings)
     .settings(akka.AkkaBuild.defaultSettings)
     .enablePlugins(BootstrapGenjavadoc)
+    .settings(
+      addCompilerPlugin(scalafixSemanticdb),
+      scalacOptions += "-Yrangepos"
+    )
```

Comment:
I don't think we can add this due to https://github.com/lightbend/genjavadoc/issues/220